### ISSUE TITLE
Analysis completed with no impact layer

### ISSUE THREAD

**Issue by lucernae** [kartoza/geosafe#528]
Some analysis like this:

https://geonode.ingc.gov.mz/geosafe/analysis/219

Has completed with no impact layer.
But, since we already implemented Task Diagnostics, when the task completed, we can be sure that the task itself returned with no errors. Which means, possibly there is something wrong with the data.

Possible cause:

- The analysis has completed but since there are no impact, the impact layer is empty.
- The hazard or exposures have some misconfiguration in the InaSAFE keywords. Some info might be doesn't match up, which causes InaSAFE doesn't found the geometry/features affected and returned no impact layer

**Comment by gubuntu** [comment]
in at least one case this is because the hazard layer has InaSAFE 5 keywords instead of 4.4

**Comment by lucernae** [comment]
Yeah, currently trying to download the layer to know the actual issue

**Comment by gubuntu** [comment]
just look at the metadata detail

**Comment by lucernae** [comment]
For this particular layer, it turns out that this layer: https://geonode.ingc.gov.mz/geosafe/analysis/219 has no metadata upload (xml).
Not sure how this happens in the first place, but I just reupload the metadata, fix the upload session for this particular layer so it can have xml file, fix the extent to be consistent with metadata from the admin page, then rerun analysis.
All analysis that uses this as its exposure layer now runs fine.
So this is just data problem. I think we can close this.

The reason why the original impact analysis return `None` is because the exposure layer doesn't have keywords (as a file), so the analysis completed (no features), but no impact layer produced (zero analysis).

I think we can close this.
